src/core/lib/address_utils/sockaddr_utils.cc

```diff
@@ -272,6 +303,10 @@ absl::StatusOr<std::string> grpc_sockaddr_to_uri(
   if (scheme == nullptr || strcmp("unix", scheme) == 0) {
```

Comment:
Seems a bit odd that the pre-existing code here is calling `grpc_sockaddr_to_uri_unix_if_possible()` if the URI scheme is unknown.  I suggest changing this to:

```
if (scheme == nullptr) {
  return absl::InvalidArgumentError("Unknown address type");
}
if (strcmp("unix", scheme) == 0) {
  return grpc_sockaddr_to_uri_unix_if_possible(resolved_addr);
}
if (strcmp("vsock", scheme) == 0) {
  return grpc_sockaddr_to_uri_vsock_if_possible(resolved_addr);
}
```

Reply:
resolved